presto-main/src/main/java/io/prestosql/operator/HttpPageBufferClient.java

```diff
@@ -605,6 +639,33 @@ public PagesResponse handle(Request request, Response response)
             }
         }
 
+        private void verifyChecksum(URI uri, long readChecksum, List<SerializedPage> pages)
+        {
+            // TODO identify our address, so that query failure reports both ends of communication
```

Comment:
cc @electrum

Reply:
I'd file an issue for this and include it in the comment